nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/content/recipe/HostedHandlers.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2018-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.repository.helm.internal.content.recipe;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+import org.sonatype.goodies.common.ComponentSupport;
+import org.sonatype.nexus.repository.view.Content;
+import org.sonatype.nexus.repository.view.Handler;
+import org.sonatype.nexus.repository.view.matchers.token.TokenMatcher.State;
+import org.sonatype.repository.helm.internal.AssetKind;
+import org.sonatype.repository.helm.internal.util.HelmPathUtils;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static org.sonatype.nexus.repository.http.HttpResponses.notFound;
+import static org.sonatype.nexus.repository.http.HttpResponses.ok;
+
+/**
+ * Helm Hosted Handlers
+ *
+ * @since 0.0.2
```

Comment:
Is it an actual version?

Reply:
Fixed 134e0fecc36375934a081e33e7ea5d90e2c92183